README.md

```diff
@@ -264,5 +264,29 @@ For example:
   RtspTcpPort = "8554"
 ```
 
+
+### CameraStatus REST API Endpoint
+Use the following query to determine the status of the camera.
+
+URL parameter:
+- **DeviceName**: The name of the camera
+
+```
+curl -X GET http://localhost:59882/api/v2/device/name/[DeviceName]/CameraStatus?InputIndex=0 | jq -r '"CameraStatus: " + (.event.readings[].value|tostring)'
+```
+
+   Example Output: 
+   ```
+    CameraStatus: 0
```

Comment:
You can paste the entire json output instead of this and point to the field that holds the json value.

Reply:
You can also point how the user can get the `InputIndex` value via `curl -X GET http://localhost:59882/api/v2/device/name/example-camera/VideoInputIndex | jq .` Cameras can have one or more of these.